src/project.ts

```diff
@@ -140,6 +151,12 @@ export function path(
   }
   result.push("projects", urn);
 
+  if (content == ProjectContent.History) {
+    result.push("history");
+  } else {
+    result.push("tree");
```

Comment:
I'd use a switch here, there will be other cases in the future.

Reply:
done in c1e2717cae73295799262c6a6b02e52d988653c4